score/liquid_icx/tests/test_integrate_liquid_icx.py

```diff
@@ -30,71 +30,111 @@ def test_score_update(self):
             self.assertEqual(self._score_address, tx_result['scoreAddress'], msg=pp.pformat(tx_result))
 
     def test_0_join_delegate_stake_fallback(self):
+        """
```

Comment:
Can we refactor `LICXTestBase` so that the flag `LOCAL_NETWORK_TEST` and the environment variables are overriden by the inheriting classes in `test_integrate_liquid_icx.py` and `test_integrate_liquid_icx_with_fake_SCORE.py`?